lib/presentation/view/search.dart

```diff
@@ -1,12 +1,459 @@
+                            filterController.resetFilters();
+                            Get.back();
+                          },
+                          child: const Icon(
+                            Icons.close,
+                            color: Colors.white,
+                          ),
+                        ),
+                      ),
+                    ],
+                  ),
+                ),
+                ListTile(
+                  title: const Text(cuisineTitle),
+                  subtitle: DropdownButton<String>(
+                    isExpanded: true,
+                    value: filterController.cuisine.value.propertyName,
+                    items: CuisineEnum.values
+                        .map(
+                          (cuisine) => DropdownMenuItem<String>(
+                            value: cuisine.propertyName,
+                            child: Text(cuisine.propertyName),
+                          ),
+                        )
+                        .toList(),
+                    onChanged: (value) {
+                      filterController.updateCuisine(
+                        CuisineEnum.values.firstWhere(
+                          (e) => e.propertyName == value,
+                        ),
+                      );
+                    },
+                  ),
+                ),
+                ListTile(
+                  title: const Text(mealTypeTitle),
+                  subtitle: DropdownButton<String>(
+                    isExpanded: true,
+                    value: filterController.mealType.value.propertyName,
+                    items: MealTypeEnum.values
+                        .map(
+                          (mealType) => DropdownMenuItem<String>(
+                            value: mealType.propertyName,
+                            child: Text(mealType.propertyName),
+                          ),
+                        )
+                        .toList(),
+                    onChanged: (value) {
+                      filterController.updateMealType(
+                        MealTypeEnum.values.firstWhere(
+                          (e) => e.propertyName == value,
+                        ),
+                      );
+                    },
+                  ),
+                ),
+                ListTile(
+                  title: const Text(maxCarbsTitle),
+                  trailing: Text(
+                    filterController.maxCarbs.value.toString(),
+                  ),
+                  subtitle: Slider(
+                    max: SlidersEnum.maxCarbs.maxValue,
+                    min: SlidersEnum.maxCarbs.minValue,
+                    value: filterController.maxCarbs.value,
+                    onChanged: (value) {
+                      filterController.updateMaxCarbs(
+                        value.floorToDouble(),
+                      );
+                    },
+                  ),
+                ),
```

Comment:
You can create a new widget, using title, trailing, max and min values for slider and also onChanged method call

Reply:
Created SliderFilter widget.